packages/ember-debug/lib/handlers.js

```diff
@@ -0,0 +1,27 @@
+import isPlainFunction from 'ember-debug/is-plain-function';
+
+export let HANDLERS = { };
```

Comment:
I'm not sure what you mean, these are global and are not changed per application or container.